olsodb/db/sqlalchemy/api.py

```diff
@@ -1,10 +1,14 @@
 import sys
+import alembic
 
-from oslo_db.sqlalchemy import enginefacade
+import alembic.migration as alembic_migration
```

Comment:
So. Alembic is required for reflecting declarative objects into db AND connect to the engine? Gosh. In sqlalchemy entire flow for creating database out of model from scratch and execute some operation is easy as:
```python

Meta = MetaData()
Base = declarative_base(metadata=Meta)

class Foo(Base)
    __tablename__ = 'foo'

    id = Column(Integer, primary_key=True)
    value = Column(String)

    def __init__(self, value):
        super(Foo, self).__init__()
        self.value = value


Session = sessionmaker()
engine = create_engine('sqlite://')
Meta.bind = engine
Meta.create_all(checkfirst=True)
session = Session()

# and then just do yout things with session
try:
    session.query(Foo).all()
    session.add(Foo('something'))
    session.commit()
except:
    session.rollback()
finally:
    session.close()
```